R/estimatr_iv_robust.R

```diff
@@ -129,6 +152,14 @@ iv_robust <- function(formula,
     warning("More regressors than instruments")
   }
 
+  fes <- !is.null(model_data[["fixed_effects"]])
```

Comment:
prefer is.formula over !is.null

Reply:
It's not a formula, and it's probably a character, but it could be numeric. I am going to stick with the `!is.null` on this one :/